operator-common/src/main/java/io/strimzi/operator/common/DefaultAdminClientProvider.java

```diff
@@ -43,61 +41,43 @@
      */
     @Override
     public Admin createAdminClient(String bootstrapHostnames, Secret clusterCaCertSecret, Secret keyCertSecret, String keyCertName) {
-        Admin ac;
-        String trustStorePassword = null;
-        File truststoreFile = null;
+        String publicKeys = null;
```

Comment:
Can we call this `trustedCertificates` rather than `publicKeys`? Remember that a certificate contains a public key, but also other information, so it's relevant that it's a cert not a key. Also it's relevant that it's _trusted_.